config/schedule.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
-every 1.day, at: "2:00 am" do # 1.minute 1.day 1.week 1.month 1.year is also supported
+env :PATH, ENV.fetch("PATH", nil)
+
+every 1.day, at: "5:00 am" do
   rake "decidim:metrics:all"
```

Comment:
only `decidim:metrics:all` and `decidim:open_data:export` cronjobs for 0.27?
I think that at least we also need `decidim:delete_data_portability_files`. As we deploy in a server, `tmp:clear` will also be good every night.